### DISCUSSION TITLE
Graph of investment liquidity

### DISCUSSION THREAD

**Discussion by sveetya** [ghostfolio/ghostfolio#907] [Ideas]
I think it would be great if you could add something as liquidity/marketability index to your investments. 

You would then know, how fast you can exchange your investments for money in case you desperately needed it. 

There could also be a graph, that would show you how liquid are your assets.

What do you guys think about it?

**Comment by dtslvr** [comment]
Hi @Marilok

Thank you for describing your idea. I have just deployed a new release where you can filter by asset class on the allocations page.

I think with a similar approach, an overview for liquidity could also be achieved. What do you think of a filter `Higher Liquidity` which combines the asset classes `CASH` and `COMMODITY` and another filter `Lower Liquidity` including the asset classes `FIXED_INCOME` and `REAL_ESTATE`?
